src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Morphisms/Constructors.jl

```diff
@@ -149,3 +149,33 @@ identity_map(P::AbsProjectiveScheme) = ProjectiveSchemeMor(P, P,
                                                         check=false
                                                        )
 
+function sub(P::AbsProjectiveScheme, I::Ideal)
+  @req base_ring(I) === homogeneous_coordinate_ring(P) "ideal must be defined in the homogeneous coordinate ring of the scheme"
+  inc = ProjectiveClosedEmbedding(P, I)
+  return domain(inc), inc
+end
```

Comment:
for `subscheme` we set the ambient space for the domain. Perhaps we should do the same here?